glideinmonitor/webserver/webserver.py

```diff
@@ -90,11 +102,19 @@ def flask_api_job_download_db(job_id):
     return send_file(api_job_file(job_id, False), as_attachment=True)
 
 
+@app.route('/api/job_download/condor_id/<factory>/<feuser>/<entry_name>/<condor_job_id>')
+@auth.login_required
+def flask_api_condorjob_download_db(factory, feuser, entry_name, condor_job_id):
+    # Download job for given condor id
+    return send_file(api_condorjob_file(factory, feuser, entry_name, condor_job_id), as_attachment=True)
+
+
 @app.route('/api/job_info/<job_guid>')
 @auth.login_required
 def flask_api_job_info(job_guid):
     # Info for a job given GUID
-    return api_job_info(job_guid, True)
+    info = api_job_info(job_guid, True)
+    return redirect("/job/" + str(json.loads(info)["ID"]), code=303)
```

Comment:
Here you redirect, should there be also a route "/api/job_info/condor_job/..." with a redirect to "/condor_job/...."

Reply:
This is to have a way to show the job information in a browser given a job_guid (reusing the html page for job). With condor_job there is no such use case IMHO because you only have one way to access the page (the condor job id)